linchpin/__init__.py

```diff
@@ -956,14 +956,14 @@ def _find_n_run_pb(self, pb_name, inv_src, console=True):
         self._get_role(pb_name)
```

Comment:
I guess this function 'get_role' should be changed? Because if pb_name is not the name of 'role' it would crash

Reply:
For example, the pb_name would be 'setup' if you run 'linchpin setup', it will raise an error

Reply:
setup is the name of the role, I changed it in this PR